website/docs/cloud-docs/users-teams-organizations/2fa.mdx

```diff
@@ -34,3 +34,11 @@ If you are an organization owner you can require all users within your organizat
 Click **Settings** in your organization to reach your organization'a settings page, then click **Authentication**.
 
 Click the button "Require two-factor". Please remember that all organization owners must have two-factor authentication on before this can be set.
+
+## Requiring Two-factor Authentication for Users with HashiCorp Cloud Platform 
+
+When you require two-factor authentication for all users and have users who [log in with their HashiCorp Cloud Platform Account](/terraform/cloud-docs/users-teams-organizations/users#log-in-with-your-hashicorp-cloud-platform-account), the required configuration depends on the linked HashiCorp Cloud account identity:
```

Comment:
maybe we can clarify this

"the required configuration for each Org member depends on their linked HashiCorp Cloud identity who is trying to access the Org."